forge-gui/res/cardsfolder/upcoming/wumpus_aberration.txt

```diff
@@ -0,0 +1,10 @@
+Name:Wumpus Aberration
```

Comment:
Pretty sure I saw this card in one of the other recent PRs.

Reply:
It was unclaimed in the project so I did it